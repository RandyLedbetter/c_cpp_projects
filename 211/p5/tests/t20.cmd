1 tests/t20.in results/t20.myout
