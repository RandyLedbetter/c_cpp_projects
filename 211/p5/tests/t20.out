registers[0] = $100
